charts/rancher-vsphere-csi/Chart.yaml

```diff
@@ -1,22 +1,22 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: vSphere CSI
-  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.24.0-0'
+  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.26.0-0'
   catalog.cattle.io/namespace: kube-system
   catalog.cattle.io/os: linux,windows
   catalog.cattle.io/permits-os: linux,windows
   catalog.cattle.io/rancher-version: '>= 2.6.0-0 < 2.7.0-0'
```

Comment:
Does this need to be updated for `v2.7.0`?

Reply:
Although it does not affect RKE2 to build its own chart based on this, I decided to remove the upper bound of the rancher version now so we do not need to update the chart again when we bring it into rancher/chart later.